pages/proposals.jsx

```diff
@@ -2,21 +2,628 @@ var React = require('react');
+    var checkbox = document.querySelector("#other-format");
+    if (checkbox.checked) {
+      var input = document.querySelector(".format-other-input");
+      var inputValid = !!input.value.trim();
+      if (!inputValid) {
+        document.querySelector("#format-other-error-message").classList.add("show");
+        valid = false;
+      }
+    } else if (!this.validate(["format-checkbox"])) {
+      valid = false;
+    }
+    if (!valid) {
+      window.location.hash = "#anchor-form";
+      return;
+    }
+    this.refs.submitButton2.getDOMNode().classList.add("waiting");
+    this.submit("email, first-name, surname, organization, other-facilitators, twitter-handle, space-radio, format-checkbox, desc-allow, desc-working, participants, outcome, languge, travel-radio, backup-space-radio".split(", "));
+  },
+  exhibitOnSubmit: function() {
+    document.querySelector("#generic-error-1").classList.remove("show");
+    var valid = this.validate("privacyPolicy, original-work, good-to-show, learn-and-reflect, desc-your-work, exhibit-link, exhibit-method, exhibit-title".split(", "));
+    if (!valid) {
+      window.location.hash = "#anchor-form";
+      return;
+    }
+    this.refs.submitButton1.getDOMNode().classList.add("waiting");
+    this.submit("email, first-name, surname, organization, other-facilitators, twitter-handle, space-radio, exhibit-title, exhibit-method, exhibit-link, desc-your-work, learn-and-reflect, good-to-show, another-space-radio".split(", "));
+  },
+  onNext: function() {
+    var valid = this.validate("email, first-name, surname, space-radio".split(", "));
+    if (!valid) {
+      window.location.hash = "#anchor-form";
+      return;
+    }
+
+    if (this.state.exhibit) {
+      this.setState({
+        section: "exhibit"
+      }, function() {
+        window.location.hash = "#anchor-top";
+      });
+    } else {
+      this.setState({
+        section: "desc"
+      }, function() {
+        window.location.hash = "#anchor-form";
+      });
+    }
+  },
+  renderStepOne: function() {
+    return (
+      <div>
+        <InputCombo errorMessage="First name is required." for="firstName" type="text" name="first-name">
+          First Name *
+        </InputCombo>
+        <InputCombo errorMessage="Surname name is required." for="surname" type="text" name="surname">
+          Surname *
+        </InputCombo>
+        <InputCombo errorMessage="Email is required." for="email" type="text" name="email">
+          Email *
+        </InputCombo>
+        <InputCombo for="organization" type="text" name="organization">
+          <div>Organisation</div>
+          <div>Please leave blank if you don't want it published.</div>
+        </InputCombo>
+        <InputCombo for="otherFacilitators" type="text" name="other-facilitators">
+          <div>Other facilitators</div>
+          <div>One per line, format is Name[single space]Twitter handle [single space]email (e.g. John Doe @j.doe j.doe@example.com)</div>
```

Comment:
@ScottDowne could we make the values commas-separated (instead of using single spaces)?